mlrun/runtimes/local.py

```diff
@@ -442,7 +442,15 @@ def exec_from_params(handler, runobj: RunObject, context: MLClientCtx, cwd=None)
         try:
             if cwd:
                 os.chdir(cwd)
-            val = handler(**kwargs)
+            # Apply the MLRun handler decorator for parsing inputs using type hints and logging outputs using log hints:
+            val = mlrun.handler(
+                inputs=(
+                    runobj.spec.inputs_type_hints
+                    if runobj.spec.inputs_type_hints
+                    else True
```

Comment:
What does `True` mean here? And what does `False` in the `outputs`? 
Also, if there are no `returns` and no `input_type_hints` then why not simply skipping this?

Reply:
As I see it, in the end we will log every result by default, with a default key like "result" or "dataset" or "model" (if multiple are returned, then "x_1", "x_2", ...) so I want the user to see that the usual "return" string result is no longer being returned and get used to that (it was not making much sense anyway, when returning objects they will simply be a string repr of the class). 

I did as you suggested but sooner or later this will be changed (as the approach I suggested here is more MLOpsy in my opinion).